blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/IOContainer.java

```diff
@@ -34,6 +38,9 @@
     private static final Configuration configuration = Configuration.getInstance();
 
     private ShardStateIO shardStateIO;
+    private MetadataIO metadataIO;
+    private TestIO testIO;
```

Comment:
I am not sure I like the TestIO class in the classes of main code. In fact, I was going to propose moving the TestIO class in the src/integration-test/java package.